pylint/checkers/dunder_methods.py

```diff
@@ -105,8 +105,6 @@
     "__floor__": "Use math.floor function",
     "__ceil__": "Use math.ceil function",
     "__enter__": "Invoke context manager directly",
```

Comment:
I think we should handle this generically by changing this datastructure (DUNDER_METHODS) to include the min version of the dunder method. Maybe:

```python
DUNDER_METHODS = {
    "3.10": {
       "__aiter__": "Use iter built-in function",
       "__anext__": "Use next built-in function",
    }
    "default" : {
        "__index__": "Use index method",
        "__round__": "Use round built-in function",
        ...
    }
}

```